internal/controller/git/git.go

```diff
@@ -145,17 +185,44 @@ type ClientOptions struct {
 	Credentials *RepoCredentials
 }
 
+const (
+	// FilterBlobless is a filter that excludes blobs from the clone.
+	// When using this filter, the initial Git clone will download all
+	// reachable commits and trees, and only download the blobs for commits
+	// when you do a Git checkout (including the first checkout during the
+	// clone).
+	//
+	// When using a blobless clone, you can still explore the commits of the
+	// repository without downloading additional data. This means that you can
+	// perform commands like `git log`, or even `git log -- <path>` with the
+	// same performance as a full clone.
```

Comment:
Did you mean to say _better_ performance than a full clone?

Reply:
The operation (`git log`, etc.) does not have better performance. It's performing the same as a full clone. The cloning operation itself _is_ however lighter. (You do less, but get the same benefits.)

Reply:
Yeah. I understood that to technically be so, but the commentary on the performance of `git log` seemed unnecessary to get the point across that this is a lighter weight way to clone, which left me wondering what exactly was being said here...

Now I think I realize what you were actually trying to do was express that the lighter weight clone doesn't at all _impair_ the performance of `git log`.

I just kinda read this wrong. All good.